src/app/visual/split-view/split-view-with-repeater-visual.component.html

```diff
@@ -3,6 +3,7 @@ <h3>Split view demo with repeater</h3>
 <div
   class="app-screenshot"
   id="screenshot-split-view-with-repeater"
+  style="height: 500px"
```

Comment:
I like that you added this here to see what impact a set height would have on the repeater. Do you think we should also point a visual test at this to catch any accidental CSS breaks to that height restriction?

Reply:
There weren't any visual tests for the repeater split view. I added them to match the non-repeater and added some "Lorem ipsum" text to that demo in order to trigger the scrolling mechanism.